saleor/settings.py

```diff
@@ -372,7 +372,7 @@ def get_host():
 TEST_RUNNER = ''
 
 ALLOWED_HOSTS = get_list(
-    os.environ.get('ALLOWED_HOSTS', 'localhost,127.0.0.1'))
+    os.environ.get('ALLOWED_HOSTS', '*'))
```

Comment:
I'm not sure what are the downsides of having a wildcard as ALLOWED_HOSTS?

Reply:
This is only fall back for dev. Downsides on prod: https://docs.djangoproject.com/en/2.1/ref/settings/#allowed-hosts

Reply:
Why do we need to change that?

Reply:
@patrys if your API will be on the different domain that client browser will restrict requests ([CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS)).

This code sets proper headers for allowed host domains. So in the scenario:
* API on api.mystore.com,
* client on mystore.com,

you need to set `ALLOWED_HOSTS` to `["api.mystore.com", "mystore.com"]` and everything works.

In addition local development with API on `localhost:8000` and client `localhost:3000` is also covered.

Reply:
Why would it be on a different domain during local development? I'm against putting `*` here as default as people will leave it as it in production mode which is a potential attack vector. CORS is in no way related to ALLOWED_HOSTS and in my opinion it should not be.

Reply:
I'd leave it as previous - it will force people to set it correctly in production without a way to leave it too open.

Reply:
I agree that we shouldn't set `*` here. I'll change that.